contribs/src/main/java/com/netflix/conductor/contribs/kafka/KafkaProducerManager.java

```diff
@@ -0,0 +1,50 @@
+package com.netflix.conductor.contribs.kafka;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.netflix.conductor.core.config.Configuration;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.common.serialization.LongSerializer;
+
+import java.util.Objects;
+import java.util.Properties;
+
+public class KafkaProducerManager {
```

Comment:
Class name contains two different functionalities. Producer and Manager. Here we are just getting producer. It should be KafkaProducer.

Reply:
This class manages Kafka producer to be used in KafkaPublish Task. Currently, this creates a new Kafka producer per call.  
 In the future, this class can be enhanced to support Kafka producer caching.